scale_build/bootstrap/utils.py

```diff
@@ -0,0 +1,17 @@
+from scale_build.utils.manifest import get_manifest
+from scale_build.utils.run import run # noqa
+from scale_build.utils.paths import BUILDER_DIR, CACHE_DIR, CHROOT_BASEDIR, HASH_DIR, TMP_DIR, TMPFS # noqa
+
+
+def get_apt_preferences():
+    return '\n\n'.join(
+        '\n'.join(f'{k}: {v}' for k, v in pref.items()) for pref in get_manifest()['apt_preferences']
+    )
+
+
+def get_cache_filename(cache_type):
+    return f'basechroot-{cache_type}.squashfs'
```

Comment:
Are `basechroot` and `cache` another names for something we already have above in this diff?